packages/quantic/cypress/e2e/next-analytics-expectations.ts

```diff
@@ -0,0 +1,144 @@
+          expect(eventMeta).to.have.property('type', 'Qna.AnswerAction');
+          expect(eventMeta.config).to.have.property(
+            'trackingId',
+            expectedTrackingId
+          );
+        })
+        .logDetail(
+          `should emit the Qna.AnswerAction event with action "${expectedEvent.action}"`
+        );
+    },
+
+    emitQnaCitationHover: (
+      expectedEvent: Qna.CitationHover,
+      expectedTrackingId: string
+    ) => {
+      cy.wait(InterceptAliases.NextAnalytics.Qna.CitationHover)
+        .then((interception): void => {
+          const eventBody = interception?.request?.body?.[0];
+          const eventMeta: EventMetadata = eventBody.meta;
+
+          expect(eventBody.answer).to.deep.equal(expectedEvent.answer);
+          expect(eventBody.citation).to.deep.equal(expectedEvent.citation);
+          expect(eventMeta).to.have.property('type', 'Qna.CitationHover');
+          expect(eventMeta.config).to.have.property(
+            'trackingId',
+            expectedTrackingId
+          );
+        })
+        .logDetail('should emit the Qna.CitationHover event');
+    },
+
+    emitQnaCitationClick: (
+      expectedEvent: Qna.CitationClick,
+      expectedTrackingId: string
+    ) => {
+      cy.wait(InterceptAliases.NextAnalytics.Qna.CitationClick)
+        .then((interception): void => {
+          const eventBody = interception?.request?.body?.[0];
+          const eventMeta: EventMetadata = eventBody.meta;
+
+          expect(eventBody.citation).to.deep.equal(expectedEvent.citation);
+          expect(eventBody.answer).to.deep.equal(expectedEvent.answer);
+          expect(eventMeta).to.have.property('type', 'Qna.CitationClick');
+          expect(eventMeta.config).to.have.property(
+            'trackingId',
+            expectedTrackingId
+          );
+        })
+        .logDetail('should emit the Qna.CitationClick event');
+    },
+
+    emitQnaLikeEvent: (
+      expectedEvent: Qna.SubmitFeedback,
+      expectedTrackingId: string
+    ) => {
+      cy.wait(InterceptAliases.NextAnalytics.Qna.SubmitFeedback.Like)
+        .then((interception): void => {
+          validateSubmitFeedbackEvent(
+            interception,
+            expectedEvent,
+            expectedTrackingId
+          );
+        })
+        .logDetail(
+          'should emit the Qna.SubmitFeedback event for the like action'
+        );
+    },
+
+    emitQnaDislikeEvent: (
+      expectedEvent: Qna.SubmitFeedback,
+      expectedTrackingId: string
+    ) => {
+      cy.wait(InterceptAliases.NextAnalytics.Qna.SubmitFeedback.Dislike)
+        .then((interception): void => {
+          validateSubmitFeedbackEvent(
+            interception,
+            expectedEvent,
+            expectedTrackingId
+          );
+        })
+        .logDetail(
+          'should emit the Qna.SubmitFeedback event for the dislike action'
+        );
+    },
+
+    emitQnaSubmitNegativeFeedbackEvent: (
```

Comment:
```suggestion
    emitQnaSubmitFeedbackEvent: (
```

Even though feedback is mostly submitted when there is a dislike, I think this name would match the name of the event better no?  (ie: Qna.SubmitFeedback)

Reply:
I see, then maybe something like: emitQnaSubmitFeedbackDetailsEvent or emitQnaSubmitFeedbackReasonEvent ? anyways I'll let you decide what is best here this is not a game breaking change in the first place.

Reply:
good idea `emitQnaSubmitFeedbackReasonEvent ` represent better this function 👍